src/cli/cli.ts

```diff
@@ -91,6 +91,12 @@ export class Cli {
                     return options.protocol == 'https';
                 }
             }, {
+                name: 'sslCertChainPath',
```

Comment:
Since this is not a common configuration, let's omit this from the cli initial setup.

Thanks